openstack/nova-hypervisor-agents/bin/multipathd

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+exec chroot /host /usr/bin/env -i PATH="/sbin:/bin:/usr/sbin:/usr/bin" "${0##*/}" "$@"
```

Comment:
Would that `"${0##*/}"` mean if I symlink the file to something else and call it, I would call any other binary from the host?

Reply:
Yes, you would., but say locking things down should more based on how the user in the container is mapped on users outside of it.